packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -1676,7 +1675,8 @@ export abstract class Chart extends Observable implements ModuleInstance, ChartS
     }
 
     private createSeries(seriesOptions: SeriesOptionsTypes): UnknownSeries {
-        const seriesInstance = seriesRegistry.create(seriesOptions.type, this.getModuleContext()) as UnknownSeries;
+        const seriesModule = ModuleRegistry.getSeriesModule(seriesOptions.type);
+        const seriesInstance = seriesModule!.create(this.getModuleContext()) as UnknownSeries;
```

Comment:
Do we need the `!` here and below?

Reply:
Yes, because `seriesModule` can also be `undefined`.
We could change the logic to throw in such cases, to simplify the types, but it's a bigger change.